front-end/src/components/Main.vue

```diff
@@ -76,6 +92,22 @@
             </div>
           </b-list-group-item>
         </b-list-group>
+
+        <!-- TODO refactor into component -->
+        <b-button-group style="width: 100%">
```

Comment:
You can use the Bootstrap helper class `w-100` instead of setting the style here.